conference.js

```diff
@@ -502,7 +502,6 @@ export default {
         let requestedVideo = false;
 
         if (!config.disableInitialGUM
-                && !options.startWithVideoMuted
```

Comment:
This sounds incorrect.

Reply:
this only creates the tracks- the equivalent startWithAudioMuted does not prevent the audio track from being created.

Reply:
Creating the track implies asking for permission, we should not be creating a video track if `startWithVideoMuted` is true.

Reply:
The  same counter-argument can be said also for audio, but I can understand why for audio we still need to create the track. In this case I'll need to create a new api command for toggling video, as the current toggleVideo api command does not create the video track (so does not behave as the video button from the prejoin/in meeting toolbar)

Reply:
Because some features rely on the track existing like "talk while muted", and muting an audio track is a very cheap operation as opposed to a video track, since it need to be re-created.

Reply:
If toggling video is an issue, we can add an optional "ensure video track" parameter, which setVideoMuted already implements.

Reply:
Yep I understand why audio track should still be created. On it with adding the new param for toggleVideo.

Reply:
👍

Reply:
undid this, also I unified the toggleVideo from api command and VideoMuteButton into a single action.